devops/steps/linux/anybuild.yml

```diff
@@ -34,22 +34,24 @@ steps:
 
     readonly abClientExe="$HOME/.local/share/Microsoft/AnyBuild/AnyBuild.sh"
 
-    env -i                                                                \
-      SECRET=$(LLVMPrincipalPassword)                                     \
-      HOME="$HOME"                                                        \
-      PATH="$PATH"                                                        \
-    "$abClientExe"                                                        \
-      --RemoteExecServiceUri $(AnyBuildEnvironmentUri)                    \
-      --NoCheckForUpdates                                                 \
-      --WaitForAgentForever                                               \
-      --DoNotUseMachineUtilizationForScheduling                           \
-      --ClientApplicationId $(LLVMPrincipalAppId)                         \
-      --ClientSecretEnvironmentVariable SECRET                            \
-      --ExperimentName "${{ parameters.BuildName }}"                      \
-      --WhyCacheMiss                                                      \
-      --WhyCacheMissOptions CacheDataStoreKey=${{ parameters.BuildName }} \
-      ${{ parameters.ExtraAnyBuildArgs }}                                 \
-      --                                                                  \
+    env -i                                                                  \
+      SECRET=$(LLVMPrincipalPassword)                                       \
+      HOME="$HOME"                                                          \
+      PATH="$PATH"                                                          \
+    "$abClientExe"                                                          \
+      --RemoteExecServiceUri $(AnyBuildEnvironmentUri)                      \
+      --NoCheckForUpdates                                                   \
+      --WaitForAgentForever                                                 \
+      --DoNotUseMachineUtilizationForScheduling                             \
+      --ClientApplicationId $(LLVMPrincipalAppId)                           \
+      --ClientSecretEnvironmentVariable SECRET                              \
+      --ExperimentName "${{ parameters.BuildName }}"                        \
+      --LogDir .                                                            \
+      --NoSessionLogDir                                                     \
```

Comment:
I also added `--LogDir . --NoSessionLogDir` to compensate for a recent breaking change (these switches don't cause the build to break; they only cause the logs to go to a different directory, because of which the pipeline won't  find them and won't upload them as artifacts when a build fails)